support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/DigitalSubscriptionCorporateRedemptionBuilder.scala

```diff
@@ -55,6 +51,7 @@ class DigitalSubscriptionCorporateRedemptionBuilder(
 
     redeemedSubcription
       .map(subscription => subscriptionData.copy(subscription = subscription))
+      .map(subscriptionData => subscribeItemBuilder.build(subscriptionData, state.salesForceContact, None, None))
```

Comment:
creating the SubscribeItem doesn't call any external services, so I moved it here rather than in the handler.